app/models/okta_redis/app.rb

```diff
@@ -23,6 +23,12 @@ def logo
       okta_response.body['_links']['logo'].last['href']
     end
 
+    def privacy_url
+      app_name = okta_response.body['label']
+      app_data = DirectoryApplication.find_by(name: app_name)
+      app_data.privacy_url
```

Comment:
This throws an error for apps in the testing environment that are not part of the Application Directory.

Reply:
I've pushed a commit that adds an if statement to line 29